File: Feitos em C/2497.c
#include <stdio.h>
int main() {
	int experience=1;
	while(2>1){
		int n,ciclos=0;
		scanf("%d",&n);
		if(n==-1){
			break;
		}
		while(n>1){
			ciclos++;
			n-=2;
		}
		printf("Experiment %d: %d full cycle(s)\n",experience,ciclos);
		experience++;
	}
	return 0;
}

File: Feitos em C Sharp/2497.cs
using System;
namespace ConsoleApp26 {
    class Program {
        static void Main(string[] args) {
			int experience = 1;
			while (true) {
				int ciclos = 0;
				int n = int.Parse(Console.ReadLine());
				if (n == -1) {
					break;
				}
				while (n > 1) {
					ciclos++;
					n -= 2;
				}
				Console.WriteLine("Experiment "+experience+": "+ ciclos + " full cycle(s)");
				experience++;
			}
		}
    }
}